hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1414,14 +1414,41 @@ message TenantInfo {
     optional string bucketPolicyGroupName = 5;
 }
 
+message TenantUserPrincipalInfo {
+    repeated string accessIds = 1;
+}
+
+// For storing extended access ID information (related to a tenant) to OM DB
+// Does not include the access ID value itself
+message ExtendedAccessIdInfo {
+    optional string tenantId = 1;
+    optional string userPrincipal = 2;
+    optional bool isAdmin = 3;
+    optional bool isDelegatedAdmin = 4;
+    repeated string roleIds = 5;
+}
+
+// For `tenant user info` to store access ID along with its tenant-related info
+message TenantAccessIdInfo {
+    optional string accessId = 1;
+    optional string tenantId = 2;
+    optional bool isAdmin = 3;
+    optional bool isDelegatedAdmin = 4;
+}
+
+message TenantUserInfo {
```

Comment:
I think we can pull this message's fields into the TenantGetUserInfoResponse.

Reply:
Done